pkgs/tools/networking/ssl-proxy/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, stdenv, buildGo118Module, fetchFromGitHub }:
+buildGo118Module rec {
+  version = "0.2.7";
+  pname = "ssl-proxy";
+
+  src = fetchFromGitHub {
+    owner = "suyashkumar";
+    repo = "ssl-proxy";
+    rev = "v${version}";
+    sha256 = "152a1vqc2rki34jwchg7bynys8sp03qzmkhasmrhvbk475s4pl3k";
+  };
+
+  vendorHash = "sha256-310K9ZSxy/OQ4HYFCcHQaj4NQwzATrOZ2YkhiSkhY5I=";
+
+  testPhase = ''
+    make test
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/suyashkumar/ssl-proxy";
+    description = "Simple single-command SSL reverse proxy with autogenerated certificates (LetsEncrypt, self-signed)";
+    longDescription = ''
+      A handy and simple way to add SSL to your thing running on a VM--be it your personal jupyter
+      notebook or your team jenkins instance. ssl-proxy autogenerates SSL certs and proxies
+      HTTPS traffic to an existing HTTP server in a single command.
+    '';
+    license = licenses.mit;
+    maintainers = [ maintaners.konst-aa ];
```

```suggestion
    maintainers = [ maintainers.konst-aa ];
```

Typo.
Should fix eval.

---

https://www.youtube.com/watch?v=iamld0fZkLM